src/session.ts

```diff
@@ -108,6 +112,11 @@ export class SessionManager implements Middleware {
         this.createStatusBarItem();
 
         this.powerShellExePath = this.getPowerShellExePath();
+        cp.exec(`${this.powerShellExePath} -v`, (error: cp.ExecException, stdout: string, stderr: string) => {
```

Comment:
So Windows PowerShell won't work like you want.  :-(
```
04-18 18:58:30 64> powershell -v
Missing argument for parameter version.
```

Reply:
Gah... Does `-version` work?

Reply:
There is the `this.versionDetails` field in session.ts.  You'd just have to wait until after it is set.  That is what we use to determine the version to display in the status bar.

Reply:
`-Version` in Windows PowerShell is to do this:

```
> powershell -version 2 -command '$PSVersionTable'

Name                           Value
----                           -----
CLRVersion                     2.0.50727.9136
BuildVersion                   6.1.7600.16385
PSVersion                      2.0
WSManStackVersion              2.0
PSCompatibleVersions           {1.0, 2.0}
SerializationVersion           1.1.0.1
PSRemotingProtocolVersion      2.1
```

(and values other than 2 are ignored)

Reply:
That's worse:
```
04-18 18:59:17 66> powershell.exe --version
At line:1 char:3
+ --version
+   ~
Missing expression after unary operator '--'.
At line:1 char:3
+ --version
+   ~~~~~~~
Unexpected token 'version' in expression or statement.
    + CategoryInfo          : ParserError: (:) [], ParentContainsErrorRecordException
    + FullyQualifiedErrorId : MissingExpressionAfterOperator
```
In v5.x, `-Version` expects you to specify a version to start.

Reply:
You can always use:
```
powershell -encodedcommand JABQAFMAVgBlAHIAcwBpAG8AbgBUAGEAYgBsAGUALgBQAFMAVgBlAHIAcwBpAG8AbgAuAFQAbwBTAHQAcgBpAG4AZwAoACkA
```
:-)

Reply:
I'd just use `this.versionDetails` that way we don't have to spin up a second PowerShell.

Reply:
That's perfect